python_tests/test_python_frontend.py

```diff
@@ -1468,5 +1468,40 @@ def fusion_func(fd: FusionDefinition) :
             self.assertEqual(torch.real(inputs[0]), nvf_out[0])
             self.assertEqual(torch.imag(inputs[0]), nvf_out[1])
 
+    def test_cat(self):
+        inputs = [
+            torch.randn(2, 4, device="cuda"),
+            torch.randn(2, 3, device="cuda"),
+            torch.randn(4, 4, device="cuda"),
+            torch.randn(0, 4, device="cuda"),
+        ]
+
+        def fusion_func(fd: FusionDefinition):
+            t0 = fd.from_pytorch(inputs[0])
+            t1 = fd.from_pytorch(inputs[1])
+            t2 = fd.from_pytorch(inputs[2])
+            t3 = fd.from_pytorch(inputs[3])
+
+            t3 = fd.ops.cat([t0, t1], 1)
+            fd.add_output(t3)
+
+            t4 = fd.ops.cat([t0, t2], 0)
+            fd.add_output(t4)
+
+            # torch.cat accepts empty tensors (size 0 in the concat dimension),
+            # which do not affect the output.
+            # The below fails with RuntimeError: mapped_id_resize != nullptr
+            # INTERNAL ASSERT FAILED at 
+            # "/opt/pytorch/nvfuser/csrc/lower_index_compute.cpp":1306
+            # t5 = fd.ops.cat([t0, t3], 0)
+            # fd.add_output(t5)
```

Comment:
@naoyam I am not sure we need to support empty tensors in `cat`. I placed this note here just to outline one difference I noticed from `torch.cat`.

Reply:
Can we open an issue to track this as well :bow: